7.9.Вложенные_циклы._Часть_2/7.9.1.py

```diff
@@ -0,0 +1,7 @@
+num = int(input())             
+count = 0                      
+for i in range(1, num + 1):    
+    for j in range(i):         
```

Comment:
Так как переменная вложенного цикла `j` не используется, называй `_`:
```suggestion
    for _ in range(i):         
```